broker_app.py

```diff
@@ -239,10 +251,11 @@ def _upload_spreadsheet(is_update=False):
     token = request.headers.get('Authorization')
     request_file = request.files['file']
     project_uuid = request.form.get('projectUuid')
+    submission_uuid = request.form.get('submissionUuid')
 
     try:
         logger.info('Uploading spreadsheet!')
-        submission_resource = spreadsheet_upload_svc.async_upload(token, request_file, is_update, project_uuid)
+        submission_resource = spreadsheet_upload_svc.async_upload(token, request_file, is_update, project_uuid, submission_uuid)
```

Comment:
The is_update param is just passed through here, and there is subsequent branching logic further down the execution sequence.
This smells of unclean design. Maybe there would be room here for the "strategy" pattern.

Reply:
I'm afraid I don't have time to implement that.